src/js/components/ac-drawer.js

```diff
@@ -5,13 +5,17 @@ import { AppBar, MenuItem, LeftNav, FontIcon } from 'material-ui';
 import { Link } from 'react-router';
 /*eslint-enable no-unused-vars*/
 
+import { connect } from 'react-redux';
```

Comment:
I forgot to move ACDrawer out of components and into containers. Before, Main connected to a bunch of stuff to support ACDrawer's dependencies. When any component called setDrawer or setSnackbar, the *entire* app had to re-render.